service-deployment/clusterconfig-example.yaml

```diff
@@ -74,7 +74,21 @@ clusterinfo:
     k8s_dashboard_addr: 10.0.3.12
     # The port of web portal server
     port: 6969
-
+  
+  # Config of Prometheus, grafana & exporter
+  prometheusinfo:
+    # The address of the prometheus to connect to
+    prometheus_addr: http://10.0.3.9:9090
+    # port for prometheus port
+    prometheus_port: 9090
+    # The address of the grafana to connect to
+    grafana_addr: http://10.0.3.9:3000
+    # port for grafana
+    grafana_port: 3000
+    # port for node exporter
+    node_exporter_port: 9100
+    # port for node gpu-exporter
+    gexporter_port: 9202
```

Comment:
as discussed, this PR only cover docker build & push process. please exclude this change from this PR. Same for the modification in service yaml.

Reply:
Keep the same with master branch now.